example/lib/timestamp.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:floor/floor.dart';
+import 'package:intl/intl.dart';
+
+@Embed()
+class Timestamp {
+  @ColumnInfo(name: 'created_at')
+  final DateTime createdAt;
+
+  @ColumnInfo(name: 'updated_at')
+  final DateTime updatedAt;
+
+  Timestamp({required this.createdAt, required this.updatedAt});
+
+  @override
+  bool operator ==(Object other) =>
+      identical(this, other) ||
+      other is Timestamp &&
+          runtimeType == other.runtimeType &&
+          createdAt == other.createdAt &&
+          updatedAt == other.updatedAt;
+
+  @override
+  int get hashCode => createdAt.hashCode ^ updatedAt.hashCode;
+
+  @override
+  String toString() {
```

Comment:
I don't really like using the toString function to for user-facing strings. I think it would be better to keep toString for technical things and introduce a new function, something like: format() to get a pretty user-facing String